2024-03-17 12:22:14,232 INFO    Thread-20 (_run_job):12728 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-03-17 12:22:14,233 INFO    Thread-20 (_run_job):12728 [wandb_setup.py:_flush():76] Configure stats pid to 12728
2024-03-17 12:22:14,234 INFO    Thread-20 (_run_job):12728 [wandb_setup.py:_flush():76] Loading settings from C:\Users\Hp\.config\wandb\settings
2024-03-17 12:22:14,234 INFO    Thread-20 (_run_job):12728 [wandb_setup.py:_flush():76] Loading settings from C:\Users\Hp\Downloads\DL_assignment\DL_PA1\CS6910_Assignment1\wandb\settings
2024-03-17 12:22:14,235 INFO    Thread-20 (_run_job):12728 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'CS6910_DL_assignment1', 'entity': 'srividhya', 'root_dir': 'C:\\Users\\Hp\\Downloads\\DL_assignment\\DL_PA1\\CS6910_Assignment1', 'run_id': 'z3vq6rre', 'sweep_param_path': 'C:\\Users\\Hp\\Downloads\\DL_assignment\\DL_PA1\\CS6910_Assignment1\\wandb\\sweep-wn7enth1\\config-z3vq6rre.yaml', 'sweep_id': 'wn7enth1'}
2024-03-17 12:22:14,235 INFO    Thread-20 (_run_job):12728 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-17 12:22:14,236 INFO    Thread-20 (_run_job):12728 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-17 12:22:14,237 INFO    Thread-20 (_run_job):12728 [wandb_init.py:_log_setup():526] Logging user logs to C:\Users\Hp\Downloads\DL_assignment\DL_PA1\CS6910_Assignment1\wandb\run-20240317_122214-z3vq6rre\logs\debug.log
2024-03-17 12:22:14,238 INFO    Thread-20 (_run_job):12728 [wandb_init.py:_log_setup():527] Logging internal logs to C:\Users\Hp\Downloads\DL_assignment\DL_PA1\CS6910_Assignment1\wandb\run-20240317_122214-z3vq6rre\logs\debug-internal.log
2024-03-17 12:22:14,238 INFO    Thread-20 (_run_job):12728 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000142AF0C57E0>
2024-03-17 12:22:14,239 INFO    Thread-20 (_run_job):12728 [wandb_init.py:init():566] calling init triggers
2024-03-17 12:22:14,239 INFO    Thread-20 (_run_job):12728 [wandb_init.py:init():573] wandb.init called with sweep_config: {'activation': 'relu', 'batch_size': 128, 'hl_size': 32, 'initialization': 'xavier', 'loss_type': 'squared_error', 'lr': 1e-05, 'n_epoch': 5, 'n_hiddenlayer': 3, 'optimizer': 'nadam', 'reg': 0}
config: {'input_size': 784, 'output_size': 10, 'n_epoch': 5, 'n_hiddenlayer': 3, 'hl': [64, 64, 64], 'reg': 0.0005, 'lr': 0.001, 'optimizer': 'momentum', 'batch_size': 64, 'initialization': 'xavier', 'loss_type': 'cross_entropy'}
2024-03-17 12:22:14,240 INFO    Thread-20 (_run_job):12728 [wandb_init.py:init():616] starting backend
2024-03-17 12:22:14,240 INFO    Thread-20 (_run_job):12728 [wandb_init.py:init():620] setting up manager
2024-03-17 12:22:14,274 INFO    Thread-20 (_run_job):12728 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-03-17 12:22:14,278 INFO    Thread-20 (_run_job):12728 [wandb_init.py:init():628] backend started and connected
2024-03-17 12:22:14,313 INFO    Thread-20 (_run_job):12728 [wandb_run.py:_config_callback():1343] config_cb None None {'activation': 'relu', 'batch_size': 128, 'hl_size': 32, 'initialization': 'xavier', 'loss_type': 'squared_error', 'lr': 1e-05, 'n_epoch': 5, 'n_hiddenlayer': 3, 'optimizer': 'nadam', 'reg': 0}
2024-03-17 12:22:14,319 INFO    Thread-20 (_run_job):12728 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-03-17 12:22:14,319 INFO    Thread-20 (_run_job):12728 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-03-17 12:22:14,319 INFO    Thread-20 (_run_job):12728 [wandb_init.py:init():720] updated telemetry
2024-03-17 12:22:15,176 INFO    Thread-20 (_run_job):12728 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-03-17 12:22:16,370 INFO    Thread-20 (_run_job):12728 [wandb_run.py:_on_init():2262] communicating current version
2024-03-17 12:22:17,162 INFO    Thread-20 (_run_job):12728 [wandb_run.py:_on_init():2271] got version response upgrade_message: "wandb version 0.16.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-03-17 12:22:17,163 INFO    Thread-20 (_run_job):12728 [wandb_init.py:init():804] starting run threads in backend
2024-03-17 12:22:17,562 INFO    Thread-20 (_run_job):12728 [wandb_run.py:_console_start():2241] atexit reg
2024-03-17 12:22:17,563 INFO    Thread-20 (_run_job):12728 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-03-17 12:22:17,563 INFO    Thread-20 (_run_job):12728 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-03-17 12:22:17,563 INFO    Thread-20 (_run_job):12728 [wandb_run.py:_redirect():2186] Redirects installed.
2024-03-17 12:22:17,565 INFO    Thread-20 (_run_job):12728 [wandb_init.py:init():847] run started, returning control to user process
2024-03-17 12:22:17,566 INFO    Thread-20 (_run_job):12728 [wandb_config.py:__setitem__():151] config set hl = [32, 32, 32] - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x00000142AF11B010>>
2024-03-17 12:22:17,567 INFO    Thread-20 (_run_job):12728 [wandb_run.py:_config_callback():1343] config_cb hl [32, 32, 32] None
2024-03-17 13:42:00,388 ERROR   Thread-9 (_heartbeat):12728 [internal_api.py:execute():373] 404 response executing GraphQL.
2024-03-17 13:42:00,389 ERROR   Thread-9 (_heartbeat):12728 [internal_api.py:execute():374] {"errors":[{"message":"could not find agent yc0kl14e during agentHeartbeat","path":["agentHeartbeat"]}],"data":{"agentHeartbeat":null}}
2024-03-17 13:42:15,112 WARNING MsgRouterThr:12728 [router.py:message_loop():77] message_loop has been closed
